exercises/diffie-hellman/tests/diffie-hellman.rs

```diff
@@ -67,6 +67,69 @@ fn test_secret_key_correct_big_numbers() {
     assert_eq!(secret, expected);
 }
 
+#[test]
+#[ignore]
+fn test_public_key_correct_biggest_numbers() {
+    // two biggest 64bit primes
+    let p: u64 = 0xF_FFF_FFF_FFF_FFF_FC5;
+    let g: u64 = 0xF_FFF_FFF_FFF_FFF_FAC;
+
+    let private_key: u64 = 0xF_FFF_FFF_FFF_FFF_FC3;
+
+    let expected: u64 = 13_281_655_733_070_877_121;
```

Comment:
You define the first two of these values in three places, and the other two in two. May as well extract them all into module-private consts.

Reply:
Done